flow/ObjectSerializer.h

```diff
@@ -61,10 +67,20 @@ class _ObjectReader {
 };
 
 class ObjectReader : public _ObjectReader<ObjectReader> {
+	friend class _IncludeVersion;
+	ObjectReader& operator>> (ProtocolVersion& version) {
+		uint64_t result;
```

Comment:
I assume the plan is to not ignore this later?

Reply:
no, this is a bug. I need to initialize the protocol version that is passed there